docs/UserGuide.md

```diff
@@ -178,9 +189,9 @@ This section details the format of the commands available in TutorBuddy. We will
 
 The following subsections will elaborate on the specific details of each TutorBuddy's command.
 
-### General
+#### 4.2.2 General
 
-#### Viewing help : `help`
+##### Viewing help : `help`
 
 Shows a message explaining how to access the help page.
```

Comment:
I think the message for the help command can be changed to "Displays a help window of the different commands available on TutorBuddy and a link to the user guide."